en/docs/concepts/oauth2/intro-oauth2.md

```diff
@@ -16,7 +16,7 @@ usable. When working with the tokens it guarantees API security by relying on Se
 data between the web server and browsers remain private. If a user can log in to a single OAuth 2.0 provider then he can
 log in to other web services without login to those web services differently.
 
-### When to choose OAuth2.0 is for which application ?
+### When to choose OAuth2.0 is for which application?
```

Comment:
Shouldn't this be "Which OAuth2.0 grant type to choose"?

Reply:
fixed